app/src/main/java/com/example/practica1/User.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.practica1
+
+class User(
```

Comment:
Esta clase tiene toda la cara de [**data class**](https://kotlinlang.org/docs/data-classes.html). Nuestra recomendación con estas clases es que normalmente sus atributos sean inmutables (val) y tenga valores por defecto (como has hecho).

```suggestion kotlin
data class User(
    val name: String = "",
    val email: String = "",
    val password: String = "",
    val image: Int = 0,
)
```